extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -241,6 +241,20 @@ export class BitrateManager {
       video.load();
     }
   }
+
+  /**
+   * Called when a video is replaced.
+   * @param {!Element} oldVideo
+   * @param {!Element} newVideo
+   */
+  videoReplaced(oldVideo, newVideo) {
```

Comment:
In terms of API I think it would be clearer and easier to maintain/use if we had `manage()` and `unmanage()`